src/components/CountDown/CountDown.tsx

```diff
@@ -0,0 +1,133 @@
+import React, {useEffect, useRef, useState} from 'react'
+import {AppState, type StyleProp, type ViewStyle, type TextStyle} from 'react-native'
+import styled from 'styled-components/native'
+
+export type CountDownProps = {
+  /*
+  init time countdown by second
+  */
+  initialSeconds: number
+  /*
+  call when count down finish
+  */
+  onFinish?: () => void
+  /*
+  container countdown view style
+  */
+  containerStyle?: StyleProp<ViewStyle>
+  /*
+  text countdown style
+  */
+  textStyle?: StyleProp<TextStyle>
+  /*
+  loop countdown
+  */
+  loop?: boolean
+  /*
+  format countdown
+  */
+  format?: 'MM:SS' | 'HH:MM:SS' | 'DD:HH:MM:SS'
+}
+const intervalTimeBySecond = 1000
```

Comment:
I think you should pass intervalTimeBySecond as a prop with default = 1000

Reply:
sure, i will change it